pyramid/request.py

```diff
@@ -214,10 +213,8 @@ def response(self):
         right" attributes (e.g. by calling ``request.response.set_cookie()``)
         within a view that uses a renderer.  Mutations to this response object
         will be preserved in the response sent to the client."""
-        registry = self.registry
-        response_factory = registry.queryUtility(IResponseFactory,
-                                                 default=Response)
-        return response_factory()
+        response_factory = _get_response_factory(self.registry)
+        return response_factory(self)
```

Comment:
Yes, I used in 1.5 IResponseFactory in order to set custom Response class. It seems there was no other way. 

How could I know that is private and how should I set custom Response class in 1.5?